2996_reflection/reflection.md

```diff
@@ -4699,31 +4876,46 @@ consteval vector<info> enumerators_of(info type_enum);
 ::: std
 ::: addu
 ```cpp
-class access_context {
+struct access_context {
+  // access context construction
+  static consteval access_context current() noexcept;
+  consteval access_context() noexcept;
+
+private:
   info $context_$; // exposition-only
-public:
-  consteval access_context();
 };
 ```
 
 [1]{.pnum} The type `access_context` is suitable for ensuring that only accessible members are reflected on.
 
 ```cpp
-consteval access_context::access_context()
+consteval access_context access_context::current() noexcept;
 ```
 
-[#]{.pnum} *Effects*: Initializes `context_` to a reflection of the function, class, or namespace scope most nearly enclosing the function call.
+[#]{.pnum} *Effects*: Initializes `$context_$` to a reflection of the function, class, or namespace scope most nearly enclosing the function call.
 
 ```cpp
-consteval bool is_accessible(info target, access_context from = {});
+consteval access_context::access_context() noexcept;
+```
+
+[#]{.pnum} *Effects*: Initializes `$context_$` to the null reflection.
+
+```cpp
+consteval bool is_accessible(
+        info target,
+        access_context from = access_context::currrent());
 ```
 
 [#]{.pnum} *Constant When*: `target` is a reflection representing a member or base class specifier of a class.
 
-[#]{.pnum} *Returns*: `true` if the class member or base class specifier represented by `target` can be named within the scope of `from.$context_$`. Otherwise, `false`.
+[#]{.pnum} Let `$C$` be the class for which `target` represents a member or base class specifier.
+
+[#]{.pnum} *Returns*: If `from.$context_$` is the null reflection, then `false`. Otherwise, if `target` represents a class member, then `true` if the member is accessible at all program points within the definition of the entity represented by `from.$context_$` when named in class `$C$` ([class.access]). Otherwise, `true` if the base class represented by `target` is accessible at all program points within the definition of the entity represented by `from.$context_$`. Otherwise, `false`.
```

Comment:
Bullets please, since this is already basically naturally there.

Reply:
Done.